src/components/Calculator.js

```diff
@@ -1,44 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
 // eslint-disable import/no-extraneous-dependencies
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends React.Component {
   // eslint-disable-next-line no-useless-constructor
```

Comment:
- [ ] please avoid turning off linters they are used to improve your code and make it more readable.